tests/test_stats.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This module implements test coverage for the stats functions in stats.py.
+"""
+from moztelemetry import stats
+
+
+def test_mann_whitney_u():
+    SAMPLE1 = {1: 5, 2: 1, 3: 4, 4: 2, 5: 3}
+    SAMPLE2 = {1: 1, 2: 5, 3: 2, 4: 4, 5: 3}
+
+    # Basic test.
+    assert stats.mann_whitney_u(SAMPLE1, SAMPLE2) == 94.5
```

Comment:
I think you should import scipy's mannwhitneyu test and compare its test statistic to yours. Could even have comparisons on some generated histograms.

Reply:
This project doesn't currently depend on scipy. Is that an ok requirement to add?

Reply:
I tried to simply add it to the setup.py similar to numpy but it requires a fortran compiler and failed. It'll be a bit more complicated to add this it seems?